nservicebus/messaging/callbacks-disclaimer.include.md

```diff
@@ -0,0 +1,2 @@
+
+WARNING: Callbacks should be only used in exceptional situations, e.g., to introduce messaging behind a synchronous API in a legacy component that can't be changed. They allow for gradually transitioning applications towards messaging. See also [When to use callbacks](/nservicebus/messaging/callbacks.md#when-to-use-callbacks) section for more information.
```

Comment:
Callbacks should only be used in exceptional situations, for example, to introduce messaging behind a synchronous API in a legacy component that can't be changed. They allow to gradually transition applications towards messaging. See the [When to use callbacks](/nservicebus/messaging/callbacks.md#when-to-use-callbacks) section for more information.

Reply:
@HindyLambrozo - I ❤️ your suggestion. Since this is already merged, do you want to create a new PR and I'll be happy to merge it.